include/boost/geometry/core/coordinate_system.hpp

```diff
@@ -96,10 +96,25 @@ struct coordinate_system
         >::type;
 };
 
-
 template <typename Geometry>
 using coordinate_system_t = typename coordinate_system<Geometry>::type;
 
+#ifndef DOXYGEN_NO_DETAIL
+namespace detail {
+
+// Short cut for coordinate system units
+template <typename Geometry>
```

Comment:
I probably missing something here but why not simply 
```
template <typename Geometry>
using coordinate_system_units_t = typename coordinate_system_units<Geometry>::type::units;
```

Reply:
That was my first version. But `XXX_t` is in general always an alias for the meta function `typename XXX::type` and I did not want to deviate from that convention. So let's keep it like that.